code/game/machinery/computer/orders/order_computer/mining_order.dm

```diff
@@ -5,11 +5,11 @@
 	icon_state = "mining"
 	icon_keyboard = null
 	icon_screen = null
+	cargo_cheaper = TRUE
 	circuit = /obj/item/circuitboard/computer/order_console/mining
 
 	cooldown_time = 10 SECONDS //just time to let you know your order went through.
-	express_cost_multiplier = 1.5
-	uses_ltsrbt = TRUE
+	express_cost_multiplier = 0.65 // multiplies cargo shipping instead 
```

Comment:
if the variable is about express costs, it should stay about express costs

Reply:
whatever you say boss